apps/argus/README.md

```diff
@@ -0,0 +1,51 @@
+# Argus
+
+Argus is a webservice that serves price updates according to the Pulse protocol.
+The webservice processes and delivers price updates to callers when permitted by the protocol.
+The service operates a keeper task that performs callback transactions for user requests.
+
+A single instance of this service can simultaneously serve price updates for several different blockchains.
+Each blockchain is configured in `config.yaml`.
+
+## Build & Test
+
+Argus uses Cargo for building and dependency management.
+Simply run `cargo build` and `cargo test` to build and test the project.
+
+## Command-Line Interface
+
+The Argus binary has a command-line interface to perform useful operations on the contract, such as
+registering a new price provider, or requesting price updates. To see the available commands, simply run `cargo run`.
+
+## Hermes Integration
+
+Argus integrates with the Hermes API to fetch price updates for fulfilling requests. When a price update request is received, Argus:
+
+1. Retrieves the request details from the blockchain
+2. Fetches the required price updates from Hermes using the `/v2/updates/price/{publish_time}` endpoint
+3. Executes the callback on the Pulse contract with the fetched price updates
+
+The Hermes client is implemented in the `keeper/hermes.rs` module and handles:
+
+- Converting price IDs to the format expected by Hermes
+- Fetching price updates from the Hermes API
+- Parsing the response and converting it to the format expected by the Pulse contract
+- Error handling and retries
+
+## Local Development
```

Comment:
We need a deployed contract for this right? If so let's add it here.

For future: it's good if we can create a fully local deployment setup (that either creates a new ETH network or forks ETH and deploys required contracts)